README.md

```diff
@@ -71,108 +73,147 @@ To compile and build the `broadinstitute/firecloud-ui` docker image, run
+Then run the tests from the newly created image. This script will render the
+`application.conf` and `firecloud-account.pem` to be used by the test container using secrets read
+from vault. Note that if you are running outside of docker you will need to generate these files
+manually (see below). From the `automation/docker` directory:
 
 ```bash
-cd automation/docker
-./run-tests.sh 4 qa <ip of FiaB> automation $(cat ~/.vault-token)
+./run-tests.sh <# of nodes> <dev or qa> <ip of FiaB> automation $(cat ~/.vault-token)
 ```
 
-### Running locally
+Using 3 nodes should provide sufficient parallelization while fitting inside the recommended 4 GB
+docker memory setting when running on a developer workstation.
+
+### Running directly (real Chrome)
 
-If you have a FiaB running and its IP configured in your `/etc/hosts`, you can run tests locally and watch them execute in a browser.
+If you have a FiaB running and its IP is configured in your `/etc/hosts`, you can run tests locally
+and watch them execute in a browser. _IMPORTANT_: Be careful about interacting with your computer
+while running tests this way because stray keyboard and mouse interactions can interfere with test
+execution. It is recommended to use this only when writing/debugging specific tests, not running the
+entire test suite. 
 
 #### Set Up
 
-First run render script to generate necessary configs:
+Before you run the tests for the first time, make sure you have chromedriver installed. On Mac OS,
+Homebrew (`brew`) is recommended:
 
 ```bash
-cd automation
-./render-local-env.sh $PWD $(cat ~/.vault-token) <"qa" or "dev">
+brew install chromedriver
 ```
 
-
-If you have a local UI running, you will need to go into `automation/src/test/resources` and edit the `baseURL` in `application.conf`:
-
-```
-baseUrl = "http://local.broadinstitute.org/"
-```
-
-
-When starting your UI, run:
+If your version of Chrome is 61 or later, check `chromedriver --version` to make sure you're up to
+at least 2.32. If not, use `brew upgrade chromedriver` to update.
+ 
+Also run the config render script. This will render the necessary `application.conf` and
+`firecloud-account.pem` for the tests. From the `automation` directory:
 
 ```bash
-FIAB=true ENV=qa ./config/docker-rsync-local-ui.sh
+./render-local-env.sh $PWD $(cat ~/.vault-token) <dev or qa>
 ```
 
-
 #### Running tests
 
 ##### From IntelliJ
 
-Edit your run config defaults for ScalaTest.
-Add this to the VM parameters: `-Djsse.enableSNIExtension=false  -Dheadless=false`
+First, you need to set some default VM parameters for ScalaTest run configurations. In IntelliJ, go
+to `Run` > `Edit Configurations...`, select `ScalaTest` under `Defaults`, and add these VM
+parameters:
+```
+-Djsse.enableSNIExtension=false -Dheadless=false
+```
 
 Also make sure that there is a `Build` task configured to run before launch.
 
+Now, simply open the test spec, right-click on the class name or a specific test string, and select
```

Comment:
is it worth mentioning where the test spec lives?
The Google Spec lives in:
`automation/src/test/scala`

And the main tests:
`automation/src/test/scala/org/broadinstitute/dsde/firecloud/test`